src/foam/nanos/crunch/ui/UserCapabilityJunctionWAO.js

```diff
@@ -87,6 +87,18 @@ foam.CLASS({
     },
     function save_(wizardlet, options) {
       var wData = wizardlet.data ? wizardlet.data : null;
+
+      // REVIEW: hotfix for minmax capability wizardlet data not being saved causing prereqs wizardlet cancelSave_ instead of save_
+      if ( foam.nanos.crunch.MinMaxCapabilityData.isInstance(wData) ) {
+        wData = wData.clone();
+        ['selectedData', 'selectedDataStable'].forEach(k => {
+          wData[k] = wData[k].map(v => {
+            if ( typeof v === 'string' ) return v;
+            return v?.id;
+          })
+        })
+      }
+
```

Comment:
Another possible fix for minmax wizardlet https://github.com/kgrgreer/foam3/pull/1943/files.

Reply:
So https://github.com/kgrgreer/foam3/pull/1943 is able to replace this change?

Reply:
Yes